docs/migration/v4-migration.md

```diff
@@ -30,7 +30,7 @@ A number of changes were made to the configuration options passed to the `Chart`
 * Time and timeseries scales use `ticks.stepSize` instead of `time.stepSize`, which has been removed.
 * `maxTickslimit` wont be used for the ticks in `autoSkip` if the determined max ticks is less then the `maxTicksLimit`.
 * `dist/chart.js` has been removed.
-* `dist/chart.min.js` has been renamed to `dist/chart.umd.js`.
+* `dist/chart.min.js` has been renamed to `dist/chart.umd.min.js`.
```

Comment:
Not sure we should edit the migration guide.

Since if you look at the latest docs but use version 4.3 this won't be correct.

We can add a ``(`dist/chart.umd.min.js` after version 4.5.0)``.

Reply:
If they're upgrading to an older version won't they be seeing that older version's migration guide?

Reply:
Yes, if you click the right link.

But even if you go to 4.5.0 this change would still be technically incorrect since we did not rename it from old to that we renamed it to what's already there and this gets added

Reply:
Would be more in favour of that one, but it wont be a dealbreaker for me.

Seems the test in both pr's fail on that it can't load `'src/chart.umd.min.js'`:
```Uncaught NetworkError: Failed to execute 'importScripts' on 'WorkerGlobalScope': The script at 'src/chart.umd.min.js' failed to load. thrown```